source/enumerate-collections.rst

```diff
@@ -265,7 +266,17 @@ Example return::
         "system.replset"
     ]
 
-Drivers MUST strip the database name from the returned collection names.
+
+
+Drivers MAY implement a MongoClient method that returns an Iterable of strings,
+each of which corresponds to a name in the collections array of the
```

Comment:
Before the `listCollections` command, drivers simply queried `system.namespaces`, so there was no results array to speak of. There also technically isn't one when the command returns a cursor, since drivers construct a cursor from that response and iterate on _that_.

I think the most straightforward language here is to simply say that the result is an iterable of strings, where each string corresponds to a collection name (without discussing whether or not we're using a command). You can certainly keep the sentence about the method name below.